src/js/user-profile/helpers.jsx

```diff
@@ -0,0 +1,59 @@
+import Raven from 'raven-js';
+
+export const formTitles = {
+  '21P-527EZ': 'Veterans pension benefits (21P-527EZ)',
+  '21P-530': 'burial benefits (21P-530)',
+  '1010ez': 'health care (10-10EZ)',
+  '22-1990': 'education benefits (22-1990)',
+  '22-1990E': 'education benefits (22-1990E)',
+  '22-1990N': 'education benefits (22-1990N)',
+  '22-1995': 'education benefits (22-1995)',
+  '22-5490': 'education benefits (22-5490)',
+  '22-5495': 'education benefits (22-5495)'
+};
+
+export const formLinks = {
+  '21P-527EZ': '/pension/application/527EZ/',
+  '21P-530': '/burials-and-memorials/application/530/',
+  '1010ez': '/health-care/apply/application/',
+  '22-1990': '/education/apply-for-education-benefits/application/1990/',
+  '22-1990E': '/education/apply-for-education-benefits/application/1990e/',
+  '22-1990N': '/education/apply-for-education-benefits/application/1990n/',
+  '22-1995': '/education/apply-for-education-benefits/application/1995/',
+  '22-5490': '/education/apply-for-education-benefits/application/5490/',
+  '22-5495': '/education/apply-for-education-benefits/application/5495/'
+};
+
+export const trackingPrefixes = {
+  '21P-527EZ': 'pensions-527EZ-',
```

Comment:
These have been delinked from the formConfigs as the link was causing the following error:
```
There was an error while starting the test runner:

ReferenceError: location is not defined
    at Object.<anonymous> (/Users/clairehsu/Projects/vets-website/src/js/common/helpers/environment.js:5:31)
    at Module._compile (module.js:573:30)
    at loader (/Users/clairehsu/Projects/vets-website/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/clairehsu/Projects/vets-website/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/clairehsu/Projects/vets-website/src/js/common/helpers/login-helpers.js:1:1)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vets-website@1.0.0 nightwatch: `nightwatch -c config/nightwatch.js "test/user-profile/e2e/user-profile.e2e.spec.js"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the vets-website@1.0.0 nightwatch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/clairehsu/.npm/_logs/2017-08-30T05_23_56_794Z-debug.log
kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vets-website@1.0.0 test:e2e: `./script/run-nightwatch.sh "test/user-profile/e2e/user-profile.e2e.spec.js"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the vets-website@1.0.0 test:e2e script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/clairehsu/.npm/_logs/2017-08-30T05_23_56_847Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vets-website@1.0.0 test: `npm run test:unit && npm run test:e2e "test/user-profile/e2e/user-profile.e2e.spec.js"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the vets-website@1.0.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/clairehsu/.npm/_logs/2017-08-30T05_23_56_877Z-debug.log
```

Reply:
🤔 How peculiar...